### PR TITLE
Add German translation in de.po

### PR DESCRIPTION
**Pull request by WebSnke**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+"Language-Team: none\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=ASCII\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: src/panelMenuButton.js:95
+msgid "Open System Monitor"
+msgstr "Open System Monitor"
```
Comment (win0err) [inline po/de.po:22]: Wrong translation: https://github.com/GNOME/gnome-system-monitor/blob/master/po/de.po#L39-L40

File: po/de.po [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+msgid "General Preferences"
+msgstr "Allgemeine Einstellungen"
+
+#: src/resources/ui/preferences.ui:14
+msgid "Idle threshold"
+msgstr "Leerlaufzeitschwellenwert"
+
+#: src/resources/ui/preferences.ui:34
+msgid "Displaying items"
+msgstr "Artikel anzeigen"
```
Comment (vain) [inline po/de.po:50]: I’d go with `Anzeigen als` here.

This is the header for a list, isn’t it? And that list shows `Character and percentage`, `Percentage only`, and `Character only`, right? So, the user selects how the CPU load is being displayed. (You might even want to call it `Show as` in English, dunno.)

File: po/de.po [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+#: src/resources/ui/preferences.ui:38
+msgid "Character and percentage"
+msgstr "Charakter und Prozentsatz"
```
Comment (vain) [inline po/de.po:54]: As I understand it, `Character` here means the little cat, right? If so, `Figur` might be a better choice than `Charakter`.

File: po/de.po [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+#: src/resources/ui/preferences.ui:39
+msgid "Percentage only"
+msgstr "Nur Prozentsatz"
+
+#: src/resources/ui/preferences.ui:40
+msgid "Character only"
+msgstr "Nur Zeichen"
```
Comment (vain) [inline po/de.po:62]: Similarly, this should probably be `Nur Figur`.

File: po/de.po (hunk 1 above)
Comment (vain) [reply po/de.po:22]: I’ve added a few comments. The rest looks fine to me. :)

Comment (WebSnke) [reply po/de.po:22]: > P.S. Let's wait a bit for a native speaker to review the PR. Thank you :-)

I am a native speaker...

File: po/de.po (hunk 3 above)
Comment (WebSnke) [reply po/de.po:54]: Oh yeah, "Figur" sounds better.

File: po/de.po (hunk 4 above)
Comment (WebSnke) [reply po/de.po:62]: Yep.

File: po/de.po (hunk 1 above)
Comment (WebSnke) [reply po/de.po:22]: Translating it with "Freie Systemüberwachung" sounds very weird to me and I think that we should stick to the English description. That's why I left it as it was.

Comment (vain) [reply po/de.po:22]: I *think* `Open System Monitor` is the label of a button or a menu item that runs `gnome-system-monitor -r` when pressed. So maybe `Systemmonitor öffnen` could work here? I’m not a GNOME user, though, I don’t know how `gnome-system-monitor` is usually called – that term should probably be used here as well.

Comment (WebSnke) [reply po/de.po:22]: Yep, "Systemmonitor öffnen" sounds good. I thought it meant "open" as in "open-source".

File: po/de.po (hunk 2 above)
Comment (win0err) [reply po/de.po:50]: [Screencast from 2023-03-14 10-44-26.webm](https://user-images.githubusercontent.com/11278181/224930381-e477086a-c96a-45db-beab-763b0eb71365.webm)

File: po/de.po (hunk 1 above)
Comment (win0err) [reply po/de.po]: > > P.S. Let's wait a bit for a native speaker to review the PR. Thank you :-)
> 
> I am a native speaker...

Oops. But actually, I mean I don't speak German, I asked a native speaker to review.

File: po/de.po (hunk 2 above)
Comment (WebSnke) [reply po/de.po]: Ah, thanks. I just changed it to "Anzeigen als" as @vain suggested.

**Comment by WebSnke** [comment]
I think it's ready to be merged.

**Comment by vain** [comment]
> I think it's ready to be merged.

Yeah, looks fine to me. 👌

**Comment by win0err** [comment]
@WebSnke, if you want, you may add yourself as a translator (add `<property name="translator-credits" />`):
https://github.com/win0err/gnome-runcat/blob/master/src/resources/ui/preferences.ui#L83
```xml
<property name="artists">Sergei Kolesnikov, Takuto Nakamura</property>
<property name="translator-credits">Sergei Kolesnikov, Your Name</property>
```

**Comment by win0err** [comment]
@WebSnke, could you please update the translation for `feature/v22` branch in new PR?

There are new lines: https://github.com/win0err/gnome-runcat/blob/feature/v22/po/de.po

**Pull request merged**